### PR TITLE
Integrity check http status fix

### PR DESCRIPTION
**Pull request by victoralfaro-dotcms**
Using correct HTTP status to implement Integrity Check and little refactoring.

### CODE REVIEW

**Review by freddyucv — APPROVED** [review]

File: dotCMS/src/main/java/com/dotcms/rest/IntegrityResource.java [hunk 1]
```diff
@@ -256,20 +235,19 @@ public Response generateIntegrityData(@Context HttpServletRequest request)  {
...
      * if ERROR, returns HttpStatus.SC_INTERNAL_SERVER_ERROR, including the error message
      *
-     * Usage: /getdata
+     * Usage: /integrityData
      *
      */
-    @POST
-    @Path("/{requestId}/status")
+    @GET
+    @Path("/{requestId}/integrityData")
```
Comment (erickgonzalez) [inline dotCMS/src/main/java/com/dotcms/rest/IntegrityResource.java:245]: is this safe? Do we need to change the call at JSP or angular?

Comment (jdotcms) [reply dotCMS/src/main/java/com/dotcms/rest/IntegrityResource.java:245]: well probably b/c it is not changing only the path, but also the verb

**Review by jdotcms — APPROVED** [review]

**Review by dsilvam — APPROVED** [review]

File: dotCMS/src/main/java/com/dotcms/rest/IntegrityResource.java (hunk 1 above)
Comment (victoralfaro-dotcms) [reply dotCMS/src/main/java/com/dotcms/rest/IntegrityResource.java:245]: This endpoint is meant to be consumed by backend http clients to sync the integrity check between dotcms instances.
Although, at `core` I didn't find any place where we consume this endpoint from a UI. I will have to ask @fmontes to check it on `core-web`.

**Comment by dsilvam** [comment]
https://github.com/dotCMS/core/issues/20965

File: dotCMS/src/main/java/com/dotcms/rest/IntegrityResource.java (hunk 1 above)
Comment (victoralfaro-dotcms) [reply dotCMS/src/main/java/com/dotcms/rest/IntegrityResource.java:245]: According to @fmontes there is no impact on `core-web`.

Comment (fmontes) [reply dotCMS/src/main/java/com/dotcms/rest/IntegrityResource.java:245]: I search for it and I don't think we're using it in core-web

**Pull request closed without merging**